src/app/workflow/info-tab/info-tab.component.html

```diff
@@ -145,6 +145,10 @@
               {{ 'descriptor_type' | mapFriendlyValue: workflow?.descriptorType }}
             </div>
           </li>
+          <div>
+            <strong matTooltip="Digital Object Identifier for all workflow versions">DOI</strong>:
+            {{ workflow && workflow.conceptDoi ? workflow.conceptDoi : 'n/a' }}
```

Comment:
my new way `(workflow || {}).conceptDoi || 'n/a'`

Reply:
Or (`workflow && workflow.conceptDoi) || 'n/a')`. :) 

What's the TS 3.7 feature?

Reply:
the way I mentioned is because it scales well when heavily nested.

so it would be `(((((layer1 || {}).layer2 || {}).layer3 || {}).layer4 || {})`
instead of `layer1 || layer1.layer2 || layer1.layer2.layer3 || layer1.layer2.layer3.layer4`

Reply:
https://github.com/microsoft/TypeScript/issues/16#issuecomment-515160784 for TS 3.7

Reply:
Fixed; used (workflow && workflow.conceptDoi) || 'n/a')